messenger/doctrine-transport.rst

```diff
@@ -0,0 +1,214 @@
+        <container xmlns="http://symfony.com/schema/dic/services"
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xmlns:framework="http://symfony.com/schema/dic/symfony"
+            xsi:schemaLocation="http://symfony.com/schema/dic/services
+                http://symfony.com/schema/dic/services/services-1.0.xsd
+                http://symfony.com/schema/dic/symfony
+                http://symfony.com/schema/dic/symfony/symfony-1.0.xsd">
+
+            <framework:config>
+                <framework:messenger>
+                    <framework:transport name="doctrine" dsn="doctrine://default?table_name=custom_table_name_for_messages" />
+                </framework:messenger>
+            </framework:config>
+        </container>
+
+    .. code-block:: php
+
+        // config/packages/messenger.php
+        $container->loadFromExtension('framework', array(
+            'messenger' => array(
+                'transports' => array(
+                    'doctrine' => array(
+                        'dsn' => 'doctrine://default?table_name=custom_table_name_for_messages',
+                    ),
+                ),
+            ),
+        ));
+
+Use the same table for different messages
+-----------------------------------------
+
+If you want to store the messages in the same table you can configure the `queue_name` option.
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        # config/packages/messenger.yaml
+        framework:
+            messenger:
+                transports:
+                    doctrine:
+                        dsn: "doctrine://default?queue_name=custom_queue"
+
+    .. code-block:: xml
+
+        <!-- config/packages/messenger.xml -->
+        <?xml version="1.0" encoding="UTF-8" ?>
+        <container xmlns="http://symfony.com/schema/dic/services"
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xmlns:framework="http://symfony.com/schema/dic/symfony"
+            xsi:schemaLocation="http://symfony.com/schema/dic/services
+                http://symfony.com/schema/dic/services/services-1.0.xsd
+                http://symfony.com/schema/dic/symfony
+                http://symfony.com/schema/dic/symfony/symfony-1.0.xsd">
+
+            <framework:config>
+                <framework:messenger>
+                    <framework:transport name="doctrine" dsn="doctrine://default?queue_name=custom_queue" />
+                </framework:messenger>
+            </framework:config>
+        </container>
+
+    .. code-block:: php
+
+        // config/packages/messenger.php
+        $container->loadFromExtension('framework', array(
+            'messenger' => array(
+                'transports' => array(
+                    'doctrine' => 'doctrine://default?queue_name=custom_queue',
+                ),
+            ),
+        ));
+
+Available options
+-----------------
+
+Options for the transport can be configure on the DSN or as options of the transport.
```

Comment:
configure -> configured 

But the sentence sounds a bit weird to me 🤔

Reply:
I re-read this and I found this weird too. The `options` repetition is the problem I think.

Maybe : "The transport can be configured via DSN or as options" ? :thinking:

Reply:
Yes, it sounds better 👍